src/index.ts

```diff
@@ -0,0 +1,57 @@
+import {NativeModules} from 'react-native';
```

Comment:
Instead of making the whole file typed which is unnecessary for simple javascript code and also creates trouble while development. Let's just export type definition for this library. Let all code be just plain javascript.

Reply:
> Instead of making the whole file typed which is unnecessary for simple javascript code and also creates trouble while development. Let's just export type definition for this library. Let all code be just plain javascript.
> 
> BTW I don't know typescript and expect lot of the users also not to. Why add extra dependencies when the simple way works.

TypeScript is great for any developer, any module update that changes the behavior of any method is automatically recognized by the user when looking at their IDE and already proposing a way to correct deprecated methods. It is simple to learn and at the same time it offers more code readability, better code auto completion, the possibility to better understand a code by looking at the limited types that any variable or function has. In my case I am passionate about TypeScript much more than pure JavaScript. In addition, any project written in TypeScript can be compiled in seconds for JavaScript, but in a JavaScript project, writing the types later for specific TypeScript users requires more work hours.

Reply:
But yes, lot of users still don't know ts

Reply:
We will export type declaration file that should do for people using typescript project.